Contacts

TrueMolni opened the pull request:
Тег <address> мав би бути в футері, і його не може бути декілька. Тому в картках я його не зробив

TrueMolni committed:
Update contacts.html

TrueMolni committed:
Update contacts.html

скелет секції "Contacts" з класами. Без стилів.

TrueMolni committed:
Update contacts.html

Додав заголовок секції

TrueMolni committed:
Update contacts.html

Приховав заголовок класом

akelarka reviewed (commented):
![image](https://user-images.githubusercontent.com/108392983/188274723-bb04787e-198e-4c52-b9a6-786398d90c4d.png)
все добре, там де підзаголовок можна зробити або все великими буквами, або все маленькими і потім додати uppercase в стилях, бо зараз одне так інше так. Це єдине, що в очі кинулось. Мерджити буду тільки зі стилями вже, бо потім буде повно конфліктів, які вручну потрібно буде вирішувати

akelarka commented:
і ще як на мене, то все ж таки **СAFE** та **FOOD TRUCK** будуть ашками, а міста звичайними пешками

TrueMolni committed:
Merge branch 'main' of https://github.com/akelarka/ice-cream-24-team-project into contacts

TrueMolni committed:
v1

TrueMolni committed:
Update _contacts.scss

v1.1

TrueMolni committed:
v1.3

Додав всі описові стилі

TrueMolni committed:
v1.4

Зроби всі стилі для різних пристроїв.
Padings на секцію не давав.
Потрібні ще класи  на кнопки.
Проблема з background-image

TrueMolni committed:
Merge branch 'main' of https://github.com/akelarka/ice-cream-24-team-project into contacts

TrueMolni committed:
v1.5

Додав класи на кнопки в contacts

TrueMolni committed:
Update _contacts.scss

TrueMolni committed:
v 1.6

Зробив абсолют на картки, перевір, будь-ласка.
